pybamm/simulation.py

```diff
@@ -256,6 +257,23 @@ def set_up_and_parameterise_model_for_experiment(self):
             )
             self.experiment_unique_steps_to_model[repr(op)] = parameterised_model
 
+        # Set up rest model if experiment has timestamps
```

Comment:
How come we need a separate model for this? Can't we use the current model and set the current to be zero like we do for a rest?

Reply:
I thought it would be more robust to have a separate model for rest, in case the user only has, for example, power driven steps (unlikely, I know).

Reply:
ah ok, yeah, makes sense